tensorboard/data/experimental/experiment_from_dev.py

```diff
@@ -74,9 +72,13 @@ def get_scalars(
         self,
         runs_filter=None,
         tags_filter=None,
-        pivot=True,
+        pivot=False,
         include_wall_time=False,
     ):
+        # NOTE(b/155980442): Import pandas early in this method, so if the
```

Comment:
Can this be a public issue reference, please? There’s nothing in the
linked bug that’s sensitive to Google.

Reply:
OK. I'll create a public issue for this and move the pandas-related changes to a separate PR.

Reply:
OK. This PR is now only about the default value of pivot.